### ISSUE TITLE
Importing files with the theme

### ISSUE THREAD

**Issue by SlavBoi420** [rdbende/Sun-Valley-ttk-theme#56]
Basically, I had to call in a file using the import module command so that I can access certain functions of it. It also has a GUI, however, when I use this theme, it throws up an error saying that winfo cannot be invoked. 

I know that this theme requires that we should not destroy the Tcl Interpreter, and that we should make Toplevel windows. But I wanted to use this in multiple other python scripts, so toplevel windows aren't an option. Any suggestions/help would be very much appreciated as I ain't much of an expert with Python. 

Edit: Basically, I seem to be unable to use `import` to import another python file, and I cannot use other ways to do so either as I wanna use the functions inside the script to be imported

TL;DR: I wanna import a script with a GUI (and one that uses this theme) into another one so that I can access its functions. But it seems like the original tcl interpreter is destroyed in the process

Feel free to ask if you have any questions :)

**Comment by rdbende** [comment]
So if I understand well, you have two `Tk` instances, one in the main file, and one in the file you want to import, right? Could you provide some example code?

**Comment by SlavBoi420** [comment]
Yeah, but I am so sorry to say this but I think my code is horribly wrong. 
So, I wanna connect this Python script with a MySQL database using `mysql.connector`. I usually have to type the username and password myself in the code, but I want the user to be able to type out their own username and password for the MySQL Server(if their password or something is different). So I tried making that file to be imported and then importing the connection function in that file into another program, but I found this highly impractical (and also didn't seem to work). 

Is it possible to do this? I could have done this with a toplevel window but then I will have to do it for all programs. Also, is it possible to show up the UI first where the user inputs the username and password, and then just use the input there for all other programs?

![image](https://user-images.githubusercontent.com/94615466/196835743-148d903d-ebba-47a6-b400-dde08002346b.png)

I hope this terrible chart thingy can help you understand what I am looking for

**Comment by rdbende** [comment]
I'm not sure if this is the same issue, but see #72 

TL;DR: You can't use multiple `tkinter.Tk` instances when using this theme.
